src/fhirtypes/ElementDefinition.ts

```diff
@@ -2176,15 +2176,25 @@ export class ElementDefinition {
   /**
    * Returns an array of an ElementDefinition's unfolded children.
    * @param {ElementDefinition} targetElement - The ElementDefinition being unfolded
+   * @param {boolean} recaptureSlices - Indicates whether or not slices should be recaptured
    * @returns {ElementDefinition[]} An array of the targetElement's children, with the IDs altered and
    * the original property re-captured.
    */
-  private cloneChildren(targetElement: ElementDefinition): ElementDefinition[] {
+  private cloneChildren(
+    targetElement: ElementDefinition,
+    recaptureSlices = true
```

Comment:
With the most recent change, this argument name is a little misleading.  Can we rename it to `recaptureSliceExtensions`?  (And don't forget to rename it in the JSDoc above too).

Reply:
Good point, fixed in https://github.com/FHIR/sushi/pull/955/commits/313f7337079fbc4b1bb726d656bcd875944c8187